kit/azure/activity-log/template/platform-module/terragrunt.hcl

```diff
@@ -19,12 +15,32 @@ dependency "organization-hierarchy" {
   config_path = "../organization-hierarchy"
 }
 
+generate "provider" {
+  path      = "provider.tf"
+  if_exists = "overwrite"
+  contents  = <<EOF
+provider "azurerm" {
+  features {}
+  skip_provider_registration = true
+  tenant_id       = "${include.platform.locals.platform.azure.aadTenantId}"
+  subscription_id = "the-id-of-your-logging-subscription"
+  client_id       = "${dependency.bootstrap.outputs.client_id}"
+  client_secret   = "${dependency.bootstrap.outputs.client_secret}"
+  }
+
+provider "azuread" {
```

Comment:
d: does activity log module need the azuread provider? i think it doesn't use it

Reply:
also: activity-log module is missing a providers.tf or versions.tf like the other modules have it to pin compatible azurerm versions